apps/prairielearn/src/tests/groupExam.test.ts

```diff
@@ -16,49 +17,57 @@ const courseInstanceUrl = baseUrl + '/course_instance/1';
 
 const storedConfig: any = {};
 
+const GROUP_EXAM_1_TID = 'exam14-groupWork';
+const GROUP_EXAM_2_TID = 'exam15-groupWorkRoles';
 const GROUP_NAME = 'groupBB';
 const GROUP_NAME_ALTERNATIVE = 'groupCC';
 
+const StudentUserSchema = z.object({
+  user_id: z.string(),
+  uid: z.string(),
+  name: z.string(),
+  uin: z.string().nullable(),
+});
```

Comment:
This is a subset of `UserSchema` that's already present in `lib/db-types`, so an equivalent way of writing this would be:

```ts
const StudentUserSchema = UserSchema.pick({
  user_id: true,
  uid: true,
  name: true,
  uin: true,
});
```

This is nice because it reuses the subtleties of the original type (e.g. that `user_id` has a more precise type than `z.string()`).